### PR TITLE
Move in changes for 1.2.4 release

### PR DESCRIPTION
**Pull request by paulcam206**
# Important
**This PR is intended to supercede #3561. It was necessary to rework the code changes to avoid an unfortunate UWP conflict.**

Many thanks to @RebeccaAnne for helping get this pulled together! 👍 

## Description
Cherry-pick changes for v1.2.4

## How Verified
TBD

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3565)

### CODE REVIEW

File: source/uwp/Renderer/lib/ActionHelpers.cpp [hunk 1]
```diff
@@ -0,0 +1,1034 @@
...
+// Licensed under the MIT License.
+#include "pch.h"
+
+#include "ActionHelpers.h"
+#include "AdaptiveImage.h"
+#include "AdaptiveRenderArgs.h"
+#include "AdaptiveShowCardActionRenderer.h"
+#include "LinkButton.h"
+
+using namespace Microsoft::WRL;
```
Comment (RebeccaAnne) [inline source/uwp/Renderer/lib/ActionHelpers.cpp]: This file is part of the xamlbuilder refactor and shouldn't be coming in, right?

Comment (paulcam206) [reply source/uwp/Renderer/lib/ActionHelpers.cpp]: indeed. I'll have it fixed shortly :)

Comment (paulcam206) [reply source/uwp/Renderer/lib/ActionHelpers.cpp]: fixed with 6fccd7d

Comment (paulcam206) [reply source/uwp/Renderer/lib/ActionHelpers.cpp]: and verified changes work in the visualizer

File: source/dotnet/Library/AdaptiveCards/AdaptiveInlinesConverter.cs [hunk 2]
```diff
@@ -35,7 +41,22 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
...
+                    if (ParseContext == null)
+                    {
+                        ParseContext = new ParseContext();
+                    }
+
+                    var adaptiveInline = JsonConvert.DeserializeObject<AdaptiveTextRun>(jobj.ToString(), new JsonSerializerSettings
+                    {
+                        ContractResolver = new WarningLoggingContractResolver(new AdaptiveCardParseResult(), ParseContext),
+                        Converters = { new StrictIntConverter() }
+                    });
```
Comment (nesalang) [inline source/dotnet/Library/AdaptiveCards/AdaptiveInlinesConverter.cs]: i think this one is missing a latest change

**Comment by nesalang** [comment]
looks good on ios & .net (concurrent changes)

File: source/dotnet/Library/AdaptiveCards/AdaptiveInlinesConverter.cs (hunk 2 above)
Comment (paulcam206) [reply source/dotnet/Library/AdaptiveCards/AdaptiveInlinesConverter.cs]: @nesalang -- can you elaborate? it does look like @shalinijoshi19 's revert of the serialization stuff impacted this code...

Comment (nesalang) [reply source/dotnet/Library/AdaptiveCards/AdaptiveInlinesConverter.cs]: LN27-29 has a the latest, and LN 49-55 had one older commit. I checked-in the fix via iteration 20

---
In reply to: [342306159](https://github.com/microsoft/AdaptiveCards/pull/3565#discussion_r342306159) [](ancestors = 342306159)

Comment (paulcam206) [reply source/dotnet/Library/AdaptiveCards/AdaptiveInlinesConverter.cs]: awesome! thanks!

**Pull request merged**